demosplan/DemosPlanCoreBundle/Controller/MiscContent/DemosPlanMiscContentController.php

```diff
@@ -469,9 +469,11 @@ public function documentsAction(Breadcrumb $breadcrumb, TranslatorInterface $tra
      */
```

Comment:
Since the method was enhanced, a short documentation would be nice.

Reply:
adjusted in 9a35ce471ecbaa9ad940ebd8954ce70f55999a12